util/webhook/webhook_test.go

```diff
@@ -149,10 +149,14 @@ func TestGitHubCommitEvent_MultiSource_Refresh(t *testing.T) {
 func TestGitHubCommitEvent_AppsInOtherNamespaces(t *testing.T) {
 	hook := test.NewGlobal()
 
-	patchedApps := make([]string, 0, 3)
+	type nameAndNamespace struct {
+		Name      string
+		Namespace string
+	}
```

Comment:
@Niksko can we just use standard `types.namespacedName` here instead of defining a new struct?
https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName